backend/btrixcloud/crawls.py

```diff
@@ -147,6 +147,13 @@ class CrawlCompleteIn(BaseModel):
     completed: Optional[bool] = True
 
 
+# ============================================================================
+class UpdateCrawl(BaseModel):
+    """Update crawl tags"""
+
+    tags: List[Optional[str]] = []
```

Comment:
I wonder if this be should be optional all, or want to require passing empty list?

Reply:
Yeah good point. Not much purpose in passing nothing for now, but eventually when we support multiple fields they should probably be optional. Making the change now.